packages/console/src/utilities/translation.ts

```diff
@@ -0,0 +1,9 @@
+import i18next from 'i18next';
+
+export const translateUnnamed = (resource: Record<string, string>) => {
+  const { languages } = i18next;
```

Comment:
any difference between accessing i18next here and `useTranslation`?

Reply:
Yes, `useTranslation` can only be used in react component, and it doesn't work when we want to construct a string with the translated result.

Reply:
e.g.:
```js
const modalContent = `You're deleting the ${translateUnnamed(resource)} social connector. Are you sure?`;
modal.display(modalContent);
```